<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Engatar Ve√≠culo</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="botaoSalvar">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(52, 235, 119);
border-color: rgb(0, 0, 0);
</string>
    </property>
    <property name="text">
     <string>Engatar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nrFrota">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="nrConjunto">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="dtEngate">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="botaoTelaDesengate">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>330</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(52, 235, 119);
border-color: rgb(0, 0, 0);
</string>
    </property>
    <property name="text">
     <string>Desengatar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka Banner</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Frota</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka Banner</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Conjunto</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka Banner</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Data desengate</string>
    </property>
   </widget>
   <widget class="QLabel" name="Fundo_2">
    <property name="geometry">
     <rect>
      <x>-390</x>
      <y>0</y>
      <width>861</width>
      <height>541</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../Images/testcaminhao (2).jpeg</pixmap>
    </property>
   </widget>
   <widget class="QLineEdit" name="nrPlacaFrota">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>PLACA</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nrPlacaConjunto">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>130</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>PLACA</string>
    </property>
   </widget>
   <zorder>Fundo_2</zorder>
   <zorder>botaoSalvar</zorder>
   <zorder>nrFrota</zorder>
   <zorder>nrConjunto</zorder>
   <zorder>dtEngate</zorder>
   <zorder>botaoTelaDesengate</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>nrPlacaFrota</zorder>
   <zorder>nrPlacaConjunto</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
